crates/core/src/subscription/subscription.rs

```diff
@@ -342,47 +353,64 @@ impl QuerySet {
     /// This is a *major* difference with normal query execution, where is expected to return the full result set for each query.
     #[tracing::instrument(skip_all)]
     pub fn eval(&self, db: &RelationalDB, tx: &Tx, auth: AuthCtx) -> Result<DatabaseUpdate, DBError> {
-        let mut database_update: DatabaseUpdate = DatabaseUpdate { tables: vec![] };
-        let mut table_ops = HashMap::new();
-        let mut seen = HashSet::new();
-
-        for SupportedQuery { expr, .. } in self {
-            if let Some(t) = expr.source.get_db_table() {
+        // evaluate each of the queries in this QuerySet in parallel
+        let ops = self
+            .par_iter()
```

Comment:
For debugging reasons I would still like to have the option to run this single threaded. Could we add a compile time flag or something that would allow that?

Reply:
Hmmm. That might be sorta tricky. Are you thinking in order to have the tracing spans be correct?

Reply:
It'll just be harder to identify issues with individual queries if they're all spread across different threads. If I can run all of them in a single thread, I can more easily see which parts are slow and and drill down into them.